README.adoc

```diff
@@ -368,32 +380,46 @@ InventoryManager.java
 include::finish/inventory/src/main/java/io/openliberty/guides/inventory/InventoryManager.java[]
 ----
 
-The [hotspot=Try file=0]`try` block that you see here is called a `try-with-resources` statement, meaning that the `childScope` object is closed at the end of the statement. It's good practice to define custom spans inside
-such statements. Otherwise, any exceptions that are thrown before the span is closed will leak the active span.
+The [hotspot=Try file=0]`try` block that you see here is called a `try-with-resources` statement, 
+meaning that the `childScope` object is closed at the end of the statement. 
+It's good practice to define custom spans inside such statements. 
```

Comment:
It's good practice to define custom spans inside such statements. 
--->
Defining custom spans inside such statements is a good practice .